css/index.css

```diff
@@ -14,3 +15,60 @@ h2{
     margin-left: 20px;
     font-size: 20px;
 }
+/* category buttons */
+.category {
+    display: flex;
+    flex-direction: row;
+    margin-top: -20%;
+    flex-wrap: wrap;
+    
+}
+#abstract-btn{
+    width: 150px;
+    height: 50px;
+    margin-left: 20px;
+    margin-top: 139px;
+    background:url("/assets/imgs/abstract.jpg") ;
```

Comment:
Fix the links!!
As you import from the stylesheet "someway no one prefer to",
You are in the css folder /PiX-Quotes/css
You need to import from /PiX-Quotes/assets/imgs
You need to add two dots in order go a folder backward
You should go like this

 background:url("../assets/imgs/abstract.jpg") ;
 
 Check this link [Dealing with files](https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/Dealing_with_files)